drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -27,10 +27,7 @@ class Application::User < Types::BaseObject
     end
 
     def name
-      names = [object.first_name, object.last_name].compact_blank
-      return "User #{object.id}" unless names.any?
-
-      names.join(' ')
+      object.full_name || "User #{object.id}"
```

Comment:
Not sure where `full_name` is coming from, but I assume a `User` class.  Does that return `nil` or a blank string if `full_name` is empty.  Maybe:
```suggestion
      object&.full_name.presence || "User #{object.id}"
```

Reply:
disregard, found implementation further down, it does a compact & presence check.